test/verify/test_avg_pooling_1d.cpp

```diff
@@ -31,11 +31,12 @@ struct test_avg_pooling_1d : verify_program<test_avg_pooling_1d>
 {
     migraphx::program create_program() const
     {
+        // pooling test with nonzero padding
         migraphx::program p;
         auto* mm = p.get_main_module();
         auto input =
-            mm->add_parameter("x", migraphx::shape{migraphx::shape::float_type, {1, 3, 5}});
-        auto op = migraphx::op::pooling{migraphx::op::pooling_mode::average, {0}, {1}, {3}};
+            mm->add_parameter("x", migraphx::shape{migraphx::shape::float_type, {1, 3, 7}});
```

Comment:
Dont change this test. Just add a new test(like `test_avg_pooling_1d_pad`) to set nonzero padding.

Reply:
Done.